### DISCUSSION TITLE
How to delete all tools?

### DISCUSSION THREAD

**Discussion by PepeGak** [ImageMagick/ImageMagick#5870] [Help]
I'm using Ubuntu 22.04 LTS and I've installed ImageMagick using `sudo apt-get install imagemagick`. With imagemagick, its tools have been installed like convert, mogrify, etc. And when I try to delete ImageMagick completely using `sudo apt-get remove imagemagick`, its tools are still on my pc and `apt` can't find them: `E: Unable to locate package mogrify` for this and others tools. How do I delete them all?

**Comment by urban-warrior** [comment]
This is not the appropriate forum for your question as it has nothing to do with ImageMagick, instead Ubuntu's  `apt-get`.  However, from the Ubuntu docs:

> Apt-get remove uninstalls the package from your system but keeps the configurations and other user settings associated with the package. When you reinstall the package, your old configurations and settings will load automatically.

whereas

> Unlike apt-get remove, apt-get purge completely removes the package from your system, along with the personalized settings and configurations.
